player.cpp

```diff
@@ -8,24 +8,26 @@ Player::Player()
 void Player::intitialize(Ui::MainWindow *ui_ptr)
 {
     ui = ui_ptr;
+    choices_map =
```

Comment:
I this this map should be a const

Reply:
ok noted

Reply:
cannot make it work right row
me thinks we should fix it later

Reply:
an issue have been created for this comment: https://github.com/DucDung-oreo/rock_paper_scissors/issues/4